defaults/main.yml

```diff
@@ -60,3 +60,48 @@ steamengine_java_base_opts:
 steamengine_java_opts:
   - "-Xmx512M"
   - "-Xms512M"
+
+# DRUPAL
```

Comment:
As there is no namespace, prefix your variables by "steamengine_drupal_".

Reply:
Still relevant

Reply:
Last variables (apparently) renamed in this PR https://github.com/STEAMULO/steamulo.steamengine/pull/32